.github/ISSUE_TEMPLATE/monthly-cyhy-environment-redeploy.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Monthly CyHy Environment Redeploy
+about: Monthly redeploy of the CyHy .nvironment
```

Comment:
There's a typo in the last word here.

Reply:
Thanks! Resolved in https://github.com/cisagov/cyhy_amis/pull/281/commits/d1ee4675a058de4e09a936e409100d64d3446279